libcloudforensics/providers/aws/internal/s3.py

```diff
@@ -74,10 +79,42 @@ def CreateBucket(
           ACL=acl,
           CreateBucketConfiguration={
               'LocationConstraint': region or self.aws_account.default_region
-          })  # type: Dict[str, Any]
+          })
       return bucket
+    except client.exceptions.BucketAlreadyOwnedByYou as exception:
+      raise errors.ResourceCreationError(
+          'Bucket {0:s} already exists: {1:s}'.format(
+              name, str(exception)),
+          __name__) from exception
     except client.exceptions.ClientError as exception:
       raise errors.ResourceCreationError(
           'Could not create bucket {0:s}: {1:s}'.format(
               name, str(exception)),
           __name__) from exception
+
+  def Put(self, s3_path, local_file) -> None:
```

Comment:
Missing type annotations for function args

Reply:
Done